### PR TITLE
feat: use frontend-plugin-framework to provide a FooterSlot

### PR DESCRIPTION
**Pull request by brian-smith-tcril**
Note: This removes the ability to use `LOGO_POWERED_BY_OPEN_EDX_URL_SVG` to set the logo in the footer. This was directly using `process.env` and therefore did not support the more robust configuration methods provided by https://github.com/openedx/frontend-platform/blob/master/src/config.js. The footer component itself supports using `LOGO_TRADEMARK_URL` to set the logo (and supports the more robust configuration methods).

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const App = ({ courseMetadata, isEnabled }) => (
       <main data-testid="main">
         <ListView />
       </main>
-      <Footer logo={process.env.LOGO_POWERED_BY_OPEN_EDX_URL_SVG} data-testid="footer" />
```
Comment (arbrandes) [inline src/App.jsx:35]: As with the other MFEs, can you please remove the var from .env.development and .env.test as well?

File: package.json [hunk 2]
```diff
@@ -25,14 +25,14 @@
...
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
-    "@edx/frontend-component-footer": "13.0.2",
     "@edx/frontend-component-header": "5.0.2",
     "@edx/frontend-platform": "7.1.0",
     "@edx/openedx-atlas": "^0.6.0",
     "@fortawesome/fontawesome-svg-core": "^1.2.36",
     "@fortawesome/free-brands-svg-icons": "^5.15.4",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.2.0",
+    "@openedx/frontend-slot-footer": "^1.0.2",
```
Comment (arbrandes) [inline package.json:35]: On all the other MFEs we're adding (or keeping) the explicit FPF dependency.  Shouldn't we do the same here?

**Review by arbrandes — APPROVED** [review]
Good to merge once comments are addressed.

File: src/App.jsx (hunk 1 above)
Comment (brian-smith-tcril) [reply src/App.jsx:35]: Done! https://github.com/openedx/frontend-app-ora-grading/compare/afdc7c7555220262b97fb65e1b26848c27943119..a5e0787d7862369047980190db6f3ce090bafa10

**Pull request merged**